ssl/ssl_lib.c

```diff
@@ -1126,8 +1125,7 @@ int SSL_add1_host(SSL *s, const char *hostname)
             ASN1_OCTET_STRING_free(ip);
 
             old_ip = X509_VERIFY_PARAM_get1_ip_asc(sc->param);
-            if (old_ip)
-            {
+            if (old_ip) {
```

Comment:
```suggestion
            if (old_ip != NULL) {
```

Reply:
Please note that this PR is a collection of commits, where each commit addresses a specific `checkpatch.pl` rule. Therefore, I would rather fix this issue in a different PR, perhaps searching the whole code base of this issue.

This rule is part of the [OpenSSL coding style](https://www.openssl.org/policies/technical/coding-style.html):
> Do not use implicit checks for numbers (not) being `0` or pointers (not) being `NULL`. For example, do not write

It does not seem to be part of the [Linux kernel coding style](https://www.kernel.org/doc/html/v4.10/process/coding-style.html), therefore I doubt it is enforced by `checkpatch.pl`.